docs/markdown/intro/README.md

```diff
@@ -239,6 +239,11 @@ Note that when importing individual components, any component aliases will **not
   </p>
 </div>
 
+Do not use the Nuxt module If you want to import individual BootstrapVue components into _specific_
+pages and/or components of your Nuxt app. Instead follow the [module bundlers](#using-module-bundlers)
+section above as well as the
+[selective import](#selective-component-and-directive-inclusion-in-module-bundlers) sections below.
```

Comment:
Because of the same reason of this change, individual component imports from `es/` are not working with nuxt on dev mode :)

Reply:
Hmmmm.

So we have a new issue now...

Reply:
Yes. Generally `es/` build is broken now.  It requires modern tooling for bundling (babel + webpack) and also is pretranspiled to a legacy code :D

Reply:
We could go back to cjs format for `es/`, but that would defeat the purpose of have it for tree-shake-ability (specifically when using named exports, as webpack can't tree shake named exports from csj files - not without a plugin, such as https://github.com/indutny/webpack-common-shake).

Reply:
I think component/level shaking would be possible CJS format as well. (Have to test more) but for users that really need that tree-shaking why not offering to use `src/` directly?

Reply:
Now, if we had named exports in the main index.js file (for all plugins, components and directives), and users imported from `import { BAlert } from 'bootstrap-vue'`, assuming webpack chose `"modules": "es/index.js"` as the import then it would work.  But we would need to have separate entry points when defining our different builds: `bootstrap-vue.esm.js`, `bootstrap-vue.js`, `bootstrap-vue.common.js`, and only have the top-level named exports in *.esm.js.

And revert the `es/` directory to the old common JS format. for backwards compatibility

Reply:
The issue with `src/` is that people need to enable babel to whitelist `bootstrap-vue/src` in their builds so that it get's transpiled.

Reply:
Currently `es/` is babel transpiled using `@babel/preset-env` with `modules: false` set.  So it is compatible with supported browsers, but still in the ES6 module format (although it is based on BootstrapVue's browserlistrc file)

Reply:
@pi0 what if the `es/` build files had the extension `.mjs` instead of `.js`, which would instruct Webpack (v4 at least) to treat the files as ESM modules (so it understands import/export)?

The default extensions for weback are (https://webpack.js.org/configuration/resolve/#resolveextensions):

```
module.exports = {
  //...
  resolve: {
    extensions: ['.wasm', '.mjs', '.js', '.json']
  }
}
```

Reply:
This still does not resolve node-externals problem. Probably we need to add extending whitelist support to Nuxt.js. But still `es/` files should not be preferred at least for nuxt users. (I'll come back to you on discord :D)